CONTRIBUTING.md

```diff
@@ -84,12 +74,22 @@ The core team will review your pull request and will either merge it, request ch
     git checkout -b <dev/branch-name>
     ```
 
-6. Make sure all dependencies are installed, then run the server:
+5. Make sure all dependencies are installed, then run the server:
```

Comment:
this step needs to come after the cd to the `docs` directory

Reply:
Sure! What was I thinking about? 😅 Thanks Derrick!